gama/configuration/classification.py

```diff
@@ -1,147 +1,46 @@
-# sourcery skip: de-morgan
-import numpy as np
+import ConfigSpace as cs
 
-from sklearn.naive_bayes import GaussianNB, BernoulliNB, MultinomialNB
-from sklearn.tree import DecisionTreeClassifier
-from sklearn.ensemble import (
-    ExtraTreesClassifier,
-    RandomForestClassifier,
-    GradientBoostingClassifier,
-)
-from sklearn.neighbors import KNeighborsClassifier
-from sklearn.svm import LinearSVC
-from sklearn.linear_model import LogisticRegression
-from sklearn.cluster import FeatureAgglomeration
-from sklearn.preprocessing import (
-    MaxAbsScaler,
-    MinMaxScaler,
-    Normalizer,
-    PolynomialFeatures,
-    RobustScaler,
-    StandardScaler,
-    Binarizer,
-)
-from sklearn.kernel_approximation import Nystroem, RBFSampler
-from sklearn.decomposition import PCA, FastICA
-from sklearn.feature_selection import (
-    SelectFwe,
-    SelectPercentile,
-    f_classif,
-    VarianceThreshold,
+from .classification_task import ClassifierConfig, PreprocessorConfig
+
+# Classifiers & Preprocessors 🚀
+
+# This script is your ticket to configuring a ConfigSpace object, teeming with
+# classifiers and preprocessors. We are diving in with the ClassifierConfig and
+# PreprocessorConfig classes to fill the configuration space with a slew of
+# hyperparameters and options.
+
+# Customise Your Space 🔧
+
+# Want just classifiers? No biggie! Just comment out or remove the PreprocessorConfig
+# setup + meta key-value & Voila! You're left with a sleek, classifier-only
+# configuration space.
+
+# Want to add more classifiers or preprocessors? Easy! Just add them to the
+# ClassifierConfig or PreprocessorConfig classes, respectively. You can even
+# add your own custom classifiers or preprocessors. Just make sure they are
+# compatible with scikit-learn's API.
+
+# Meta-Parameters 📝
+
+# The meta-parameters are the "estimators" and "preprocessors" keys in the
+# configuration space. These are used to identify the classifiers and preprocessors
+# by the internal system. They are not hyperparameters, and should not be
+# changed, except by advanced users. If you do change them, make sure to change
+# the corresponding values in the current configuration space, i.e. in ClassifierConfig
+# and PreprocessorConfig.
+
+# 👩‍💻👨‍💻 Happy configuring, and may your machine learning models shine!
```

Comment:
Please include this information in the documentation. https://github.com/openml-labs/gama/blob/master/docs/source/advanced_guide/configuration.inc is probably the correct file, unless you have other suggestions.

Reply:
Great ! Will be done in January, following the full review.